### PR TITLE
added navbar

### LINKED ISSUE #6: [UI] create nav-bar for website
NO nav-bar 

Create Nav-bar

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabh1908@gmail.com>

**Description**

This PR fixes #6 

**Notes for Reviewers**

**[Signed commits](https://github.com/TemplateRun/TemplateRun.io/blob/main/CONTRIBUTING.md)**

- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/general/navigation/styles.css [hunk 1]
```diff
@@ -1,202 +1,165 @@
...
+  }
+}
+
+/*
+=============== 
+Navbar
+===============
+*/
+nav {
+  background: #210248;
```
Comment (warunicorn19) [inline src/sections/general/navigation/styles.css:117]: use theme color instead

File: src/sections/general/navigation/styles.css [hunk 2]
```diff
@@ -1,202 +1,165 @@
...
+  display: block;
+  padding: 0.5rem 1rem;
+  transition: var(--transition);
+  font-family: Aileron;
+  font-style: normal;
+  font-weight: 250;
+  line-height: 36px;
+}
+.links a:hover {
+  color: #9AA4ec;
```
Comment (warunicorn19) [inline src/sections/general/navigation/styles.css:152]: Theme colour

File: src/sections/general/navigation/styles.css [hunk 3]
```diff
@@ -1,202 +1,165 @@
...
 .home-link:hover {
-    color: var(--navbar-text-color-focus);
-}
-
-.navbar-link {
-    justify-content: center;
-    width: 100%;
-    padding: 0.4em 0.8em;
-    border-radius: 5px;
+    color: #9AA4ec;
```
Comment (warunicorn19) [inline src/sections/general/navigation/styles.css]: same

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Comment by warunicorn19** [comment]
@debo19 yesterday we discussed that in mobile view the nav-bar would come out from either the left or the right side and like it is right now.

**Comment by warunicorn19** [comment]
Also font doesn't look good.

**Comment by debjani1413** [comment]
@debo19 I may be wrong but Ig it isn't responsive for all the mobile views??

**Comment by warunicorn19** [comment]
@debjani1413 actually that's because of the footer and the other contents, they are not mobile responsive but the nav-bar seems to be.

**Comment by debjani1413** [comment]
@warunicorn19 oh yes, that may be the case, alright then, no issues @debo19 🥇

**Comment by debo19** [comment]
@warunicorn19 @debjani1413 @sayantan1413 Let me know if this structure looks fine. I'll do changes to the color (use theme) then.

**Comment by warunicorn19** [comment]
@debo19 good stuff, merging this Right now, let's track the themes in a different PR.

**Pull request merged**